public/app/themes/clarity/inc/pagination.php

```diff
@@ -1,30 +1,35 @@
 <?php
 use MOJ\Intranet\Agency;
 
-function get_pagination($type, $category_id = false)
+function get_pagination($type, $category_id = false, $post_per_page = 10)
 {
     $oAgency      = new Agency();
     $activeAgency = $oAgency->getCurrentAgency();
-
-    /*
-    * A temporary measure so that API calls do not get blocked by
-    * changing IPs not whitelisted. All calls are within container.
-    */
-    $siteurl = 'http://127.0.0.1';
-
-    $post_per_page = 'per_page=10';
-    $current_page  = '&page=1';
-    $agency_name   = '&agency=' . $activeAgency['wp_tag_id'];
-    $category_name = ( ! empty($category_id) ? '&news_category=' . $category_id : '' );
-
-    $response = wp_remote_get($siteurl . '/wp-json/wp/v2/' . $type . '/?' . $post_per_page . $current_page . $agency_name . $category_name);
-
-    if (is_wp_error($response)) {
-        return;
-    }
-
-    $pagetotal = wp_remote_retrieve_header($response, 'x-wp-totalpages'); ?>
-
+    $current_page  = 1;
+
+    $args = [
+      'numberposts' => $post_per_page,
+      'post_type' => 'posts' ? 'post' : $type,
+      'post_status' => 'publish',
+      'tax_query' => [
+        'relation' => 'AND',
+        [
+          'taxonomy' => 'agency',
+          'field' => 'term_id',
+          'terms' => $activeAgency['wp_tag_id']
+        ],
+        ...($category_id ? [
```

Comment:
It is sometimes uncertain what the ... (splat) operator is doing, which could make the code complex to read and debug in future.

Maybe; could a comment be left to explain the working or indeed, move the ternary out of the default array and push the values onto the array, if the category_id is set?

You're call 👍